tests/integration/profiles/five-safes-crate/test_5src_1_responsible_project.py

```diff
@@ -101,6 +101,53 @@ def test_5src_responsible_project_member_not_organization():
     )
 
 
+# ---- SHOULD warns tests
+
+
+def test_5src_responsible_project_member_and_agent_affiliation_no_intersection():
+    """
+    Test a Five Safes Crate where none of the organisations that are members of the
+    Responsible Project appear in the Requesting Agent's affiliations (violates the
+    'Intersection with agent affiliations' SHACL warning).
+    """
+    sparql = (
+        SPARQL_PREFIXES
+        + """
+        DELETE {
+            ?agent schema:affiliation ?oldAff .
+        }
+        INSERT {
+            ?agent schema:affiliation <#missing-affiliation> .
+            <#missing-affiliation> a schema:Organization .
+        }
+        WHERE {
+            ?action a schema:CreateAction ;
+                    schema:agent ?agent .
+            ?agent a schema:Person ;
+                   schema:memberOf ?project ;
+                   schema:affiliation ?oldAff .
+            ?project schema:member ?org2 .
+
+            FILTER NOT EXISTS { ?project schema:member <#missing-affiliation> }
+        }
+        """
```

Comment:
Suggesting a slight simplification as it took me a while to wrap my head around this! (changes `?org2` to `?oldAff` and removes the FILTER - the filter condition should never occur given our test crates, right?)

```suggestion
    sparql = (
        SPARQL_PREFIXES
        + """
        DELETE {
            ?agent schema:affiliation ?oldAff .
        }
        INSERT {
            ?agent schema:affiliation <#missing-affiliation> .
            <#missing-affiliation> a schema:Organization .
        }
        WHERE {
            ?action a schema:CreateAction ;
                    schema:agent ?agent .
            ?agent a schema:Person ;
                   schema:memberOf ?project ;
                   schema:affiliation ?oldAff .
            ?project schema:member ?oldAff .
        }
        """
```